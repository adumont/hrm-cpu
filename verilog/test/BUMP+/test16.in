22 c3
45 f5
06 95
49 14
30 94
12 ba
49 52
23 1f
57 85
45 27
11 64
21 55
31 93
56 29
56 64
45 f5
23 4d
05 f4
32 00
51 c1
21 92
35 e3
01 f5
08 b0
01 51
30 51
56 c3
13 d3
25 ae
36 e1
59 f6
19 92
29 74
39 93
23 05
43 1e
07 2f
39 2f
11 31
53 a0
29 24
45 13
57 65
48 2c
49 04
41 14
15 44
