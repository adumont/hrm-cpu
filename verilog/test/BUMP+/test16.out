c4
f6
96
15
95
bb
53
20
86
28
65
56
94
2a
65
f6
4e
f5
01
c2
93
e4
f6
b1
52
52
c4
d4
af
e2
f7
93
75
94
06
1f
30
30
32
a1
25
14
66
2d
05
15
45
